docs/content/privacy.md

```diff
@@ -53,6 +53,10 @@ Users are advised to use social media platforms wisely and communicate / engage
 
 This website may use social sharing buttons which help share web content directly from web pages to the social media platform in question. Users are advised before using such social sharing buttons that they do so at their own discretion and note that the social media platform may track and save your request to share a web page respectively through your social media platform account.
 
+### Use of Google User Data ###
```

Comment:
I don't really want to alter the privacy policy like this as potentially I'd have to put in an exception for every single cloud provider and I haven't needed it for getting any of the app reviews I've had done with rclone (lots!) so far.

Reply:
This happened to me yesterday when I submitted the Rclone app to review. I assume I might be not alone here. I agree with your point about every single cloud provider, but without it seems not possible to pass the Google screening process. Have they adjusted it recently? I can attach their message because it seems to be templated.

I can rewrite this part as the 'Use of Cloud API User Data' and provide general wording without being Google-specific while putting the link to Google's Policy to Resources & Further Information.

Reply:
> seems not possible to pass the Google screening process. Have they adjusted it recently? I can attach their message because it seems to be templated

They must have changed it, I've been though it with Google 4 times.

Can you attach the message, that would be interesting

> I can rewrite this part as Use of Cloud API User Data and provide general wording without being Google-specific while putting the link to Google's Policy to Resources & Further Information.

That sounds like a good idea. Then we can add links for other providers as necessary.

Reply:
Thank you for your response. Here is the screenshot if the message (by some reason, GitHub showed an error when I tried to upload a PDF).

> The privacy policy and in-product privacy notifications clearly describe the way your application accesses, uses, stores, or shares Google user data. 

<img width="449" alt="Google-Drive-OAuth" src="https://user-images.githubusercontent.com/40397/87220715-f4da7d80-c36e-11ea-92c0-d7da694ccc88.png">

I will provide an updated wording very soon. Could you please check it as I am not a native speaker of English?